packages/wallets/src/features/cfd/modals/CTraderSuccessModal/CTraderSuccessModal.tsx

```diff
@@ -26,29 +27,35 @@ const CTraderSuccessModal = ({ displayBalance, isDemo, walletCurrencyType }: TCT
     if (isMobile) {
         return (
             <ModalStepWrapper
-                renderFooter={() => <CTraderSuccessModalButtons hide={hide} isDemo={isDemo} />}
+                renderFooter={() => (
+                    <CTraderSuccessModalButtons createdAccount={createdAccount} hide={hide} isDemo={isDemo} />
+                )}
                 title={' '}
             >
                 <CFDSuccess
                     description={description}
                     displayBalance={cTraderAccounts?.find(account => account.login)?.formatted_balance}
                     marketType='all'
                     platform='ctrader'
-                    renderButton={() => <CTraderSuccessModalButtons hide={hide} isDemo={isDemo} />}
+                    renderButton={() => (
+                        <CTraderSuccessModalButtons createdAccount={createdAccount} hide={hide} isDemo={isDemo} />
+                    )}
```

Comment:
I'm not sure why this is here because this is `mobile` and the buttons were already rendered up above in `renderFooter ` 🤔 but I'll leave it here for now, since I don't know how to fully test this feature.

```suggestion
                    renderButton={() => (
                        <CTraderSuccessModalButtons createdAccount={createdAccount} hide={hide} isDemo={isDemo} />
                    )}
```